ui/core/src/model/calculations.ts

```diff
@@ -40,13 +40,17 @@ export const CALCULATIONS_CONFIG: Readonly<Record<CalculationType, CalculationCo
     label: 'Last',
     description: 'Last value',
   },
+  FirstNumber: {
+    label: 'First number',
+    description: 'First numeric value',
+  },
   LastNumber: {
     label: 'Last number',
     description: 'Last numeric value',
   },
   Mean: {
-    label: 'Mean',
-    description: 'Average value',
+    label: 'Average',
+    description: 'Average value excluding nulls',
```

Comment:
Renaming per discussion with designers. 

Clarifying the exclusion of nulls to be explicit for the end user. Previously, this would return a `NaN` when there were `null` values, which probably isn't what we wanted.